nixos/modules/services/web-apps/shiori.nix

```diff
@@ -78,7 +108,8 @@ in {
         ProtectKernelTunables = true;
 
         RestrictNamespaces = true;
-        RestrictAddressFamilies = [ "AF_INET" "AF_INET6" ];
+        # Prevents shiori from connecting from UNIX socket.
+        # RestrictAddressFamilies = [ "AF_INET" "AF_INET6" ];
```

Comment:
```suggestion
        RestrictAddressFamilies = [ "AF_INET" "AF_INET6" "AF_UNIX" ];
```

Reply:
There is always something new to learn! made a rebase with these changes :)